app/enterprise/2.4.x/property-reference.md

```diff
@@ -223,8 +223,10 @@ which must specify an Nginx configuration template. Such a template uses the
 the given Kong configuration, before being dumped in your Kong prefix
 directory, moments before starting Nginx.
 
-The default template can be found at:
-https://github.com/kong/kong/tree/master/kong/templates. It is split in two
+The default template for Kong Gateway OSS can be found at:
+https://github.com/kong/kong/tree/master/kong/templates. The default template for
+Kong Gateway Enterprise can be found by entering the following command on the system
+running your Kong instance: `find / -type d -name "templates" | grep kong`. It is split in two
```

Comment:
```suggestion
The default template for
Kong Gateway Enterprise can be found by entering the following command on the system
running your Kong instance: `find / -type d -name "templates" | grep kong`. It is split in two
```

From Li's comments, it sounds like you can't use the OSS template for Enterprise, right? So I'd suggest removing it from here.

Reply:
ah, ok - yeah I was thinking the doc was shared with OSS but of course it isn't since it is in the enterprise folder - will fix